Supplier doesn't initialize on Java 11

Updated jdk from 10 to 11. 
Updated test-data-supplier to 1.8.2
Updated aspectj to 1.9.2

After launching testng test the follwing error occurs

```
AspectJ Internal Error: unable to add stackmap attributes. Unsupported class file major version 55
java.util.ServiceConfigurationError: org.testng.ITestNGListener: io.github.sskorol.core.DataProviderTransformer Unable to get public no-arg constructor
	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:581)
	at java.base/java.util.ServiceLoader.getConstructor(ServiceLoader.java:672)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1232)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1264)
	at java.base/java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1299)
	at java.base/java.util.ServiceLoader$3.hasNext(ServiceLoader.java:1384)
	at org.testng.TestNG.addServiceLoaderListeners(TestNG.java:967)
	at org.testng.TestNG.initializeConfiguration(TestNG.java:906)
	at org.testng.TestNG.initializeEverything(TestNG.java:996)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
Caused by: java.lang.VerifyError: Expecting a stackmap frame at branch target 45
Exception Details:
  Location:
    io/github/sskorol/core/DataProviderTransformer.supplyParallelData(Lorg/testng/ITestContext;Lorg/testng/ITestNGMethod;)Ljava/util/Iterator; @19: invokestatic
  Reason:
    Expected stackmap frame at this location.
  Bytecode:
    0000000: 2b3a 042c 3a05 b200 9c2a 2a19 0419 05b8
    0000010: 008c 4eb8 0092 2db6 0096 2a2b 2cb6 0002
    0000020: b600 03b9 0004 0100 3a07 a700 0f3a 06b8
    0000030: 0092 2db6 0099 1906 bfb8 0092 2db6 0099
    0000040: 1907 b0                                
  Exception Handler Table:
    bci [19, 45] => handler: 45

	at java.base/java.lang.Class.getDeclaredConstructors0(Native Method)
	at java.base/java.lang.Class.privateGetDeclaredConstructors(Class.java:3138)
	at java.base/java.lang.Class.getConstructor0(Class.java:3343)
	at java.base/java.lang.Class.getConstructor(Class.java:2152)
	at java.base/java.util.ServiceLoader$1.run(ServiceLoader.java:659)
	at java.base/java.util.ServiceLoader$1.run(ServiceLoader.java:656)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.ServiceLoader.getConstructor(ServiceLoader.java:667)
	... 10 more
```

--- a/build.gradle
+++ b/build.gradle
@@ -3,11 +3,11 @@ import se.bjurr.gitchangelog.plugin.gradle.GitChangelogTask
 plugins {
     id "java-library"
     id "java"
-    id "org.sonarqube" version "2.6.2"
+    id "org.sonarqube" version "2.7"
     id "com.jfrog.bintray" version "1.8.4"
-    id "net.researchgate.release" version "2.7.0"
+    id "net.researchgate.release" version "2.8.0"
     id "com.ferranpons.twitterplugin" version "1.0.2"
-    id "se.bjurr.gitchangelog.git-changelog-gradle-plugin" version "1.56"
+    id "se.bjurr.gitchangelog.git-changelog-gradle-plugin" version "1.60"
     id "com.github.ben-manes.versions" version "0.20.0"
 }
 
@@ -20,7 +20,7 @@ ext {
     projectUrl = "https://github.com/sskorol/test-data-supplier"
     moduleName = 'io.github.sskorol.testdatasupplier'
 
-    aspectjVersion = '1.9.2.RC2'
+    aspectjVersion = '1.9.2'
 }
 
 apply from: "${gradleScriptDir}/bintray.gradle"
@@ -30,9 +30,6 @@ apply from: "${gradleScriptDir}/maven-publish.gradle"
 sourceCompatibility = JavaVersion.VERSION_11
 
 repositories {
-    maven {
-        url "http://maven.springframework.org/milestone"
-    }
     jcenter()
     mavenLocal()
 }
@@ -45,20 +42,20 @@ configurations {
 
 dependencies {
     agent "org.aspectj:aspectjweaver:${aspectjVersion}"
-    compile('org.jooq:joor:0.9.9',
+    compile('org.jooq:joor:0.9.10',
             'org.testng:testng:6.14.3',
-            'one.util:streamex:0.6.7',
-            'io.vavr:vavr:0.9.2',
+            'one.util:streamex:0.6.8',
+            'io.vavr:vavr:0.10.0',
             "org.aspectj:aspectjrt:${aspectjVersion}",
             'org.reflections:reflections:0.9.11',
             'org.apache.commons:commons-csv:1.6',
             'com.google.code.gson:gson:2.8.5',
-            'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.9.7',
-            'com.fasterxml.jackson.core:jackson-databind:2.9.7'
+            'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.9.8',
+            'com.fasterxml.jackson.core:jackson-databind:2.9.8'
     )
     testCompile 'org.assertj:assertj-core:3.11.1'
-    jacoco group: 'org.jacoco', name: 'org.jacoco.ant', version: '0.8.2', classifier: 'nodeps'
-    jacocoRuntime group: 'org.jacoco', name: 'org.jacoco.agent', version: '0.8.2', classifier: 'runtime'
+    jacoco group: 'org.jacoco', name: 'org.jacoco.ant', version: '0.8.3', classifier: 'nodeps'
+    jacocoRuntime group: 'org.jacoco', name: 'org.jacoco.agent', version: '0.8.3', classifier: 'runtime'
 }
 
 sourceSets.main.output.classesDir = new File(buildDir, "classes/main")

